bin/vscode-setup

```diff
@@ -25,7 +25,11 @@ rm -rf server/temp
 
 BUILD=$(
          cat <<END
-<build>
+<properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+    <build>
```

Comment:
Done. This was originally dedented since the perl replacement below didn't account for leading whitespace. I've updated that replacement so that it also captures leading whitespace for the line.